server/api/router.go

```diff
@@ -42,6 +42,7 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	rootRouter.HandleFunc("/api/v1/schedulers", schedulerHandler.List).Methods("GET")
 	rootRouter.HandleFunc("/api/v1/schedulers", schedulerHandler.Post).Methods("POST")
 	rootRouter.HandleFunc("/api/v1/schedulers/{name}", schedulerHandler.Delete).Methods("DELETE")
+	rootRouter.HandleFunc("/api/v1/schedulers/pause", schedulerHandler.PauseOrResume).Methods("POST")
```

Comment:
I am wondering if we can reuse the API for scheduler config? @nolouch 

Regard `DelayUntil` as a configuration option and update by the scheduler handler.

Reply:
Thanks for your review, i am a little confused, how can i change shedulers' config?
1. add `DelayUntil` field in every scheduler's config.
2. add update config http handler for every scheduler like `scatter_range.go`.
3. judge whether it is ok when call `AllowScehdule`.
Or
1. add `DelayUntil` field in every scheduler's config.
2. add `SetDelayUntil` and `GetDelayUntil` in scheduler interface.
3. use `SetDelayUntil` and `GetDealyUntil` when calls `"/api/v1/schedulers"` post httphandler to change scheduler's config.
4. judge whether it is ok when call `AllowScehdule`

i am not familiar with the codes, so i am  afraid that i misunderstand some codes.

Reply:
Sorry for confusing you. I realized it is indeed complex to use configuration for schedulers. You can keep your way.

A suggestion about the API, how about as follows?

```
POST /api/v1/schedulers/{name}
{"delay": 10}
```

`name` can be the schedulers' name or "all" here.